aiosmtpd/docs/smtp.rst

```diff
@@ -187,9 +187,11 @@ return any status.  Both methods are called synchronously.
 ``handle_exception(error)``
     If implemented, this method is called when any error occurs during the
     handling of a connection (e.g. if an ``smtp_<command>()`` method raises an
-    exception).  The exception object is passed in.  Note that as part of the
-    ``SMTP`` dialog, if an exception occurs, a 500 code will be returned to
-    the client.
+    exception).  The exception object is passed in.  This method may return
+    a status string, such as ``'542 Internal server error'``.  If the method
+    returns ``None``, a 500 code will be returned to the client.  If the method
+    raises an exception, the exception raised by ``handle_exception`` will be
+    logged, and a 500 code will be returned to the client.
```

Comment:
Minor formatting comments.  I generally don't wrap "None" in backquotes.  Technically it is a symbol so should be monospace fonted, but it's so darn prevalent it's just a PITA to do.  So I don't. :)

Functions on the other hand should always have trailing parens, e.g. ``handle_exception()``.